### DISCUSSION TITLE
Doubt about the consideration of creating a Nirsoft module

### DISCUSSION THREAD

**Discussion by conexioninversa** [EricZimmerman/KapeFiles#729] [Ideas]
Hi,
I am making an automation with the great KAPE tool for the security forces of my country (Spain), they need to obtain the passwords saved in the browser in the acquisition and subsequent analysis.

I have seen that they have not created the WebBrowserPassView module and I wonder if they really see it as necessary to include it as a module (I already have it done) since perhaps showing passwords is not something that you have in mind with KAPE. Does it sound good or do I discard the idea of including it as a module?

Thanks
Pedro.

**Comment by EricZimmerman** [comment]
sure, nothing wrong with another module. we do have one for usbdeview you can use as a reference

**Reply by AndrewRathbun** [reply]
I want as many NirSoft Modules as possible but the problem I've run into is that the syntax for some of the tools isn't flexible enough to have a Module made for it. It's been easily over a year since I've last tried making one for tools that don't already have one but maybe it's time to give it another go. That being said, if you already have one working that's not in the repo, submit it! Let us know if you need help.

**Comment by EricZimmerman** [comment]
the new variables in 1.3 might help
